App/components/modals/NotificationModal.tsx

```diff
@@ -0,0 +1,86 @@
+import { useNavigation } from '@react-navigation/core'
+import { StackNavigationProp } from '@react-navigation/stack'
+import React, { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Modal, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+
+import { Colors } from '../../theme'
+
+import { Button, Title } from 'components'
+import { HomeStackParams } from 'types/navigators'
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: Colors.background,
+  },
+  childContainer: {
+    flexGrow: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  buttonContainer: {
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+  },
+  iconContainer: {
+    width: '100%',
+    flexDirection: 'row',
+    justifyContent: 'flex-end',
+  },
+  iconButton: {
+    padding: 20,
+    paddingVertical: 28,
+  },
+})
+
+interface Props {
```

Comment:
Would it matter if these were named `NotificationModalProps` for typescripts help (autocomplete). Maybe it makes no differences if all props are typed as `Props`?